src/rpc/msgpack/message.c

```diff
@@ -1,469 +0,0 @@
-/**
```

Comment:
this commit also deletes deprecated tests (see commit  fb6b70aa8f22bfb4e9b758cfaa7d152af28050b2).